apps/relayer/src/app.controller.spec.ts

```diff
@@ -111,7 +111,7 @@ describe('AppController', () => {
       })
 
       const input: DistributedBlindedPepperDto = {
-        e164Number: PHONE_NUMBER,
+        blindedPhoneNumber: await BLINDED_PHONE_NUMBER,
```

Comment:
Not sure why this `await` is here

Reply:
`getBlindedPhoneNumber` is async now as it's using the implementation in `@celo/contractkit/lib/identity/odis/phone-number-identifier`. This bubbles up through to this function.

Reply:
Ah lol that's a Promise. Ok it's fine but there's something slightly off putting about have a Promise stored in a constant. I'd rather have this as a helper function or smth but yeah we can leave it as is, ma bad!

Reply:
Ah yeah, that is strange looking. Just fixed it.